components/navbar.tsx

```diff
@@ -38,33 +60,55 @@ export default function Navbar() {
                         {session ? (
                             <>
                                 <Link href="/new-update"><a className="up-button small primary mr-4 hidden sm:block">Post new update</a></Link>
+                                <button onClick={() => setTheme(theme === 'dark' ? 'light' : 'dark')} className="up-button text">
+                                    <FiMoon/>
+                                </button>
                                 {notificationsData && notificationsData.notifications && (
-                                    <button className="mr-4 px-2 h-10 relative up-hover-button">
-                                        <FiBell/>
-                                        {notificationsData.notifications.length > 0 && (
+                                    <>
+                                    <MoreMenu

```

Comment:
this shouldn't be a MoreMenu, it should be a hover button just like the profile image. please change it back and implement other changes you made without changing it from being a hover button